[2025-08-19T09:17:38.899+0000] {task_context_logger.py:104} ERROR - Detected zombie job: {'full_filepath': '/opt/airflow/dags/data_audit_dag.py', 'processor_subdir': '/opt/airflow/dags', 'msg': "{'DAG Id': 'data_audit_dag', 'Task Id': 'validate_audit_task', 'Run Id': 'manual__2025-08-19T09:00:59.680857+00:00', 'Hostname': '37fc8edea13d', 'External Executor Id': '413f0e39-d387-41f5-a975-fd719190023f'}", 'simple_task_instance': <airflow.models.taskinstance.SimpleTaskInstance object at 0x74dd71850140>, 'is_failure_callback': True} (See https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/tasks.html#zombie-undead-tasks)
